imqsauth/config.go

```diff
@@ -75,13 +73,12 @@ func (x *Config) IsContainer() bool {
 
 func (x *Config) GetHostname() string {
 	if x.hostname == "" {
-		if x.HostnameFile != "" {
-			hostname_b, err := ioutil.ReadFile(x.HostnameFile)
-			if err != nil {
-				hostname_b, _ = ioutil.ReadFile(filepath.Join(filepath.Dir(x.lastFileLoaded), x.HostnameFile))
-			}
-			x.hostname = strings.TrimSpace(string(hostname_b))
+		hostname_b, exists := os.LookupEnv("IMQS_HOSTNAME_URL")
```

Comment:
I am not sure about this, but it would be better if we check x.hostname is not empty and then return it early. Something like 
```
	if x.hostname != "" {
		return x.hostname
	} 
```

Reply:
Agreed.